lib/src/crypto/util.rs

```diff
@@ -14,7 +14,7 @@ pub fn format_fingerprint<S: AsRef<str>>(fingerprint: S) -> String {
 /// Check whether two fingerprints match.
 pub fn fingerprints_equal<S: AsRef<str>, T: AsRef<str>>(a: S, b: T) -> bool {
     !a.as_ref().trim().is_empty()
-        && a.as_ref().trim().to_uppercase() == b.as_ref().trim().to_uppercase()
+        && a.as_ref().trim().to_uppercase().contains(&b.as_ref().trim().to_uppercase())
```

Comment:
What do you think, would [`starts_with`](https://doc.rust-lang.org/std/primitive.str.html#method.starts_with) work as well?

Because then I expect `A0B1C2 # comment` to still match `A0B1C2`.

I'd prefer it to be as strict as possible when we can.

Sorry for my late reply on this.

Reply:
wouldn'startwith` fail the new format that start with 0x<ID>...?